### PR TITLE
Merge Flutter client from original repo...

### PR DESCRIPTION
**Pull request by patniemeyer**

### CODE REVIEW

**Comment by advayDev1** [comment]
- [ ] Remove PDF and asset pngs except logo
- [ ] Waiting on confirmation of how page_view_indicator is pulled in by @theswerd

File: client/flutter/lib/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                .push(MaterialPageRoute(builder: (c) => WhoMythBusters()))),
+        PageButton(
+            title: S.of(context).travelAdvice,
+            onPressed: () => Navigator.of(context)
+                .push(MaterialPageRoute(builder: (c) => TravelAdvice()))),
+        PageButton(
+            title: S.of(context).shareTheApp,
+            lightColor: true,
+            onPressed: () => Share.share(
+                'Check out the official COVID-19 GUIDE App https://preview.whoapp.org/menu')),
```
Comment (advayDev1) [inline client/flutter/lib/home_page.dart]: Not blocking check-in this has to go away

Comment (patniemeyer) [reply client/flutter/lib/home_page.dart]: hm.  easily changed if you have a suggestion for tonight...

Comment (advayDev1) [reply client/flutter/lib/home_page.dart]: That URL would be a url to the app itself on the appropriate app store.

We have no idea what the bundle id is for Apple yet for example.

File: client/flutter/android/app/src/main/kotlin/com/example/WHOFlutter/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.example.WHOFlutter
```
[the remark this answers is not in the document]
Comment (patniemeyer) [reply client/flutter/android/app/src/main/kotlin/com/example/WHOFlutter/MainActivity.kt]: So this automatically generated hook is where the Flutter plugins register their native components.  More generally this is the place where we might end up adding a small amount of native code in the future if needed.

[the remark this answers is not in the document]
Comment (patniemeyer) [reply client/flutter/android/app/src/main/kotlin/com/example/WHOFlutter/MainActivity.kt]: Ah, I see the com.example domain was mistakenly left in there making it appear that this is example code.  I will fix.

**Pull request merged**